change/@microsoft-fast-ssr-f38ce9ee-10a7-4f57-8e4f-ae7af978d36b.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "feat: add request storage to support per-request context in ssr",
```

Comment:
Just another note on abbreviations, I know SSR is server-side-rendering, but some people may not, so should we avoid abbreviating in things that end up in change logs.